src/util/defaults.ml

```diff
@@ -228,6 +228,15 @@ let _ = ()
       ; reg Debugging "dbg.warn_with_context" "false" "Keep warnings for different contexts apart (currently only done for asserts)."
       ; reg Debugging "dbg.regression"      "false" "Only output warnings for assertions that have an unexpected result (no comment, comment FAIL, comment UNKNOWN)"
       ; reg Debugging "dbg.test.domain"     "false" "Test domain properties"
+      ; reg Debugging "dbg.warn.behavior"        "true"  "undefined behavior warnings"
+      ; reg Debugging "dbg.warn.integer"        "true"  "integer (Overflow, Div_by_zero) warnings"
+      ; reg Debugging "dbg.warn.cast"        "true"  "Cast (Type_mismatch(bug) warnings"
+      ; reg Debugging "dbg.warn.race"        "true"  "Race warnings"
+      ; reg Debugging "dbg.warn.array"        "true"  "Array (Out_of_bounds of int*int) warnings"
+      ; reg Debugging "dbg.warn.unknown"        "true"  "Unknown (of string) warnings"
+      ; reg Debugging "dbg.warn.debug"        "true"  "Debug (of string) warnings"
+      ; reg Debugging "dbg.warn.may"        "true"  "Enable or disable may warnings"
+      ; reg Debugging "dbg.warn.must"        "true"  "Enable or disable must warnings"
```

Comment:
It probably makes sense to have a separate category for warnings instead of shoving them all under debugging.

Reply:
It is now a separate category, but unfortunately just `warn` is already used in `Std` so I named the option `warn_filter`.

Reply:
`warn` should not be needed as often, so it could be renamed to `warn_at` or sth similar which is more descriptive anyway.

Reply:
alright, I renamed `warn` to `warn_at` and put all the options for our warnings into just `warn` instead of the `warn_filter`

Reply:
Seems to me like the options could even be `warn.at` and `warn.filter` also in the warning options category.

Reply:
Maybe `warn.on` since `warn.filter` can be confused as filter out or only those.